pgxn/neon/communicator/src/neon_request.rs

```diff
@@ -174,6 +181,27 @@ pub struct CGetPageVRequest {
     pub dest: [ShmemBuf; MAX_GETPAGEV_PAGES],
 }
 
+#[repr(C)]
+#[derive(Copy, Clone, Debug)]
+pub struct CReadSlruSegmentRequest {
+    pub request_id: u64,
+    pub slru_kind: SlruKind,
+    pub segment_number: u32,
+    pub request_lsn: CLsn,
+    /// Must be a null-terminated C string containing **absolute**
+    /// file path.
+    pub destination_file_path: [std::ffi::c_char; libc::PATH_MAX as usize],
```

Comment:
PATH_MAX is large. That really blows up the size of these request slots I'm afraid. See comments on `NeonIORequestSlot`

What's the max size we need? I think a `CGetPageVRequest` is somewhere around 20-30 bytes. If we can get away with less than that here, then this won't increase the overall size of the enum. I think the maximum path will look something like `pg_multixact/members/1234`.

If that doesn't work out, we'll need to encode the path in a smarter way. The `slru_kind` actually tells you the directory name already, so we could rely on that.

Reply:
We can just pass a pointer of `Shmem` pointing to the bounce buff, which is already 8192 (higher than PATH_MAX) anyway.

Reply:
Pushed that commit.